assets/src/scripts/components/hydrograph/index.js

```diff
@@ -34,16 +35,22 @@ const controlDisplay = function(elem, showElem) {
     elem.attr('hidden', showElem ? null : true);
 };
 
+/*
+ * Renders the hydrograph on the node element using the Redux store for state information. The siteno, latitude, and
+ * longitude are required parameters. All others are optional and are used to set the initial state of the hydrograph.
+ */
 export const attachToNode = function (store,
                                       node,
                                       {
                                           siteno,
-                                          parameter,
+                                          latitude,
```

Comment:
Now we are passing in the location to the hydrograph rendering function. This requires that the location is fetched in the graph-server prior to rendering. This makes it explicit where the time zone is fetched rather than extracting the location from the IV service result and then fetching it.